### DISCUSSION TITLE
When shall we meet to prepare DDD Europe?

### DISCUSSION THREAD

**Discussion by aatwi** [murex/mikado-testbuilders-kata#15] [General]
Hello @mengdaming , @cadichris , @pdalesme, 

@philou  and I started preparing for our session at DDD Europe. As you are going to animate the session with us, we would like to have you part of this preparation. 

When would be a good time for you to meet every week for 30 mins (Lunch Time, End of Day, Morning) and which day of the week? 

Also, we have created a Trello board to follow-up on the progress. You can find it [here](https://trello.com/b/JTsZ3O5y/preparation).

**Comment by pdalesme** [comment]
Hi @aatwi, 

My ordered preferences : 
End of day
Lunch time
Morning

Any day you want.

Thanks.

**Comment by CadiChris** [comment]
Hi guys !

Thanks for reaching out to us.

For me : 
**Monday** : lunch time
**Tuesday** : lunch time / end of day
**Wednesday** : lunch time
**Thursday** : lunch time
**Friday** : lunch time

**Comment by mengdaming** [comment]
Here are my preferences:

- Monday: lunch time / end of day
- Tuesday: end of day
- Wednesday: end of day
- Thursday: XXX
- Friday: lunch time / end of day

**Comment by philou** [comment]
I've sent you an invite for Tuesdays from 18:00 to 18:30. It seems to be everybody's first choice

**Reply by CadiChris** [reply]
Could we make it 20:00 to 20:30... ? 
Sorry I know it's late, but my wife has her own shop in Bordeaux, which closes at 19:00 and I'm in charge of our 14 months-old daughter at 18.

**Comment by philou** [comment]
Hi @cadichris

It's ok for me, even if the time is not ideal for me. We can move tomorrow's meeting to 20 and then reschedule to Monday lunchtime? WDYT?

Philippe

**Reply by CadiChris** [reply]
That sounds great! Thanks a lot, can't wait to start the sessions tomorrow!

**Comment by philou** [comment]
I have an impediment for Mondays lunchtime from 12 to 12:30, would 13:30 to 14:00 be ok for you?

**Reply by mengdaming** [reply]
fine with me

**Reply by pdalesme** [reply]
OK for me
